Gemfile

```diff
@@ -1,8 +1,16 @@
 raise "Ruby versions less than 2.2.2 are unsupported!" if RUBY_VERSION < "2.2.2"
+
+source 'https://rubygems.org'
+
 #
 # VMDB specific gems
 #
 
+gem "manageiq-gems-pending", ">0", :require => 'manageiq/gems/pending', :git => "https://github.com/bdunne/manageiq-gems-pending.git", :branch => "master"
+# Modified gems for gems-pending.  Setting sources here since they are git references
+gem "handsoap", "~>0.2.5", :require => false, :git => "https://github.com/ManageIQ/handsoap.git", :tag => "v0.2.5-4"
```

Comment:
Oh good 👀 

The FYI was more about there being this oddity for this particular gem where it will have to be defined both in manageiq-gems-pending (once merged and officially split off) and here.